workflow-ui/core-android/src/androidTest/java/com/squareup/workflow1/ui/container/ViewStateCacheTest.kt

```diff
@@ -172,13 +167,26 @@ internal class ViewStateCacheTest {
     }
   }
 
+  private val fakeFactory = ScreenViewFactory<NamedScreen<*>>(
+    { _, _, _ -> TODO() }, { _, _, _ -> }
```

Comment:
Hehe this looks silly and I love it

Reply:
Oops, I meant to make that a bit more explicit. :)